kong/include/kong/pluginsocket.proto

```diff
@@ -328,7 +328,7 @@ service Kong {
     rpc Router_GetRoute(google.protobuf.Empty) returns (Route);
     rpc Router_GetService(google.protobuf.Empty) returns (Service);
 
-    rpc Service_SetUpstream(String) returns (google.protobuf.Empty);
```

Comment:
I jumped the gun a little bit - does it work with only `Bool` ? The general PDK says that there are 2 values returned: (bool, string) - https://docs.konghq.com/gateway/latest/plugin-development/pdk/kong.service/#kongserviceset_upstreamhost